src/Parsers/ExpressionListParsers.cpp

```diff
@@ -1992,7 +1995,15 @@ std::unique_ptr<Layer> getFunctionLayer(ASTPtr identifier, bool is_table_functio
     else if (function_name_lowercase == "grouping")
         return std::make_unique<OrdinaryFunctionLayer>(function_name_lowercase, allow_function_parameters_);
     else
-        return std::make_unique<OrdinaryFunctionLayer>(function_name, allow_function_parameters_);
+    {
+        bool has_database_name = false;
+        if (const auto *compound_identifier  = identifier->as<ASTIdentifier>())
```

Comment:
ya doesnt have to be compound, it can be identifier as well , the check is for whether it is compound or not. 
Since we already have identifier variable, I changed this one to `ast_identifier`